### DISCUSSION TITLE
Building for Gamecube Controller works in system menus but not in any game.

### DISCUSSION THREAD

**Discussion by WattWizard** [darthcloud/BlueRetro#79] [Q&A] [answered]
I've tried with both the universal and the GC specific files but the results are always the same.  I have also tried with different Bluetooth controllers with the same results.  The most curious thing is that if I boot to the system menu and connect a controller it works great.  I put a disk in and boot it using the controller from the system menu.  The controller immediately becomes unresponsive, I can the reset the console back into the system menu without turning it off and the controller works again.  I have researched and I can't seem to find anyone with similar issues.  Anyone have any idea what I have messed up?  Thanks for any help.  I am using the HiLetgo ESP-WROOM-32 Dev board if it helps at all.

**Comment by darthcloud** [comment, marked as answer]
Any pictures of your build? Need to clearly see connections.

**Reply by WattWizard** [reply]
I was looking at the wiring diagram one last time as I was taking pictures and noticed that I had controller port 1 & 2 reversed.  Fixing that fixed the issue.  Thanks for having me look at it closer.

**Reply by darthcloud** [reply]
Good to know, wouldn't tough about it. But make sense since all 4 controller work in menu and some game might not.
